modules/end-user-guide/partials/con_support-for-theia-based-ides.adoc

```diff
@@ -0,0 +1,62 @@
+[id="con_support-for-theia-based-ides_{context}"]
+= Theia-based IDEs
+
+This section describes how to provide a custom IDE, based on Eclipse Theia framework.
+
+To use a Theia-based IDE in {prod} as an editor, you need to prepare three main components:
+
+* docker image with your IDE
+* Che editor descriptor file - `meta.yaml`
+* replace an editor in a Devfile
```

Comment:
I don't think you need this bullet point if it's described in the `Procedure` section below